projects/control-service/projects/pipelines_control_service/src/main/java/com/vmware/taurus/service/deploy/DataJobsSynchronizer.java

```diff
@@ -59,25 +70,48 @@ public DataJobsSynchronizer(
    */
   public void synchronizeDataJobs() throws ApiException {
```

Comment:
The method synchronizeDataJobs() is starting to do a lot. Consider breaking it into smaller, more descriptive methods

Reply:
Refactored